tests/integration/claims/test_claim_commands.py

```diff
@@ -1526,6 +1531,124 @@ async def test_repost(self):
         self.assertEqual(searched_repost['amount'], '0.42')
         self.assertEqual(searched_repost['signing_channel']['claim_id'], spam_claim_id)
 
+    async def test_repost_with_extensions(self):
+        tx = await self.channel_create('@goodies', '1.0')
+        goodies_claim_id = self.get_claim_id(tx)
+        tx = await self.channel_create('@spam', '1.0')
+        spam_claim_id = self.get_claim_id(tx)
+
+        # StringMap is defined in a stand-alone file, and the serialized form
+        # of the file is much smaller than that of claim.proto.
+        self.assertEqual(166, len(StringMapMessage.DESCRIPTOR.file.serialized_pb))
+
+        m0 = DescriptorMessage()
+        m0.descriptor = StringMapMessage.DESCRIPTOR.file.serialized_pb
+        ext0 = AnyMessage()
+        ext0.Pack(m0, type_url_prefix='')
+        ext0 = StreamExtension('descriptor', ext0)
+        self.assertEqual("ext.Descriptor", ext0.message.TypeName())
+
+        # create descriptor claim
+        tx = await self.stream_create(
+            f'{StringMapMessage.DESCRIPTOR.name}', '1.0', channel_name='@goodies',
+            extensions={ext0.schema: ext0},
+        )
+        string_map_claim = self.get_claim_id(tx)
```

Comment:
Illustrative of submitting a Descriptor as claim to the blockchain. To be removed.